activities/member-relations/collect-information.md

```diff
@@ -0,0 +1,33 @@
+---
+type: Resource
+---
+
+# What information to collect about members for member relations 
+
+## Introduction
+
+This resource outlines the information that Foundation for Public Code staff use to map member relations.
+
+The information collected is used to structure and keep track of our ongoing relationships with members. It is based on our [mission](../../organization/mission.md) and the first iterations of the [founding member proposal template](founding-membership-proposal.md). In line with our communication principles, it is concise and hypertextual where possible. 
+
+As information collected may include personal or sensitive information, this is not public but stored in a private and secure environment.
+
+## Information collected 
+
+* Background:
+   * History: `Provide a short outline of the organizations' history with open and collaborative code, including for example notable success stories, failures, or strong political movements`
+   * Aligned projects: ```Provide a short description of notable public-purpose open and collaborative software and policy```
+   * Opportunity projects: ```Provide a short description of notable software and policy with strong potential to become more open, collaborative and/or public-purpose```
+   * Additional comments: ```Provide a short description of other initiatives not directly related to, but adjacent to our mission```
+* Relationship to the Foundation for Public Code
+   * Decision makers: ```Outline the top two or three budget holders, policy makers and managers who decide on Foundation membership```
+   * Internal champions: ```Outline the top two or three internal champions who can help promote our mission and act as ambassadors internally```
+   * Internal skeptics: ```Outline the top two or three internal stakeholders who are skeptical of our mission, for whom we can develop and test our value proposition mapping and assumptions```
+   * Code owners: `Outline the top two or three internal stakeholders who own/manage code bases that are, or might be, stewarded by the Foundation`
+* Concrete value propositions and next steps
+   * ```Include a title for the value proposition that is actionable and descriptive, outlining how potential activities with the Foundation can solve a problem or create value for the member - such as "steward codebase X to help it be reused internationally"```
+      * Value proposition: ```Briefly outline the main value proposition - problem solved or value created - for the member, other members, the Foundation or the general ecosystem of this action.```
+      * Effort: ```If possible, provide a sense of estimated effort required (including staff time, contributors, financial etc) and any dificulties you can anticipate``` 
```

Comment:
```suggestion
      * Effort: ```If possible, provide a sense of estimated effort required (including staff time, contributors, financial etc) and any dificulties you anticipate``` 
```